services/user/Branding/service/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+#[psibase::service(name = "branding")]
+mod service {
+    use async_graphql::{Object, SimpleObject};
+    use psibase::{
+        anyhow, check, get_sender, get_service, serve_content, serve_graphiql, serve_graphql,
+        serve_simple_ui, store_content, Fracpack, HexBytes, HttpReply, HttpRequest, SingletonKey,
+        Table, ToSchema, WebContentRow,
+    };
+    use serde::{Deserialize, Serialize};
+
+    #[table(name = "NetworkNameTable")]
+    #[derive(Fracpack, ToSchema, SimpleObject, Serialize, Deserialize, Debug)]
+    pub struct NetworkName {
+        pub name: String,
+    }
+
+    impl NetworkName {
+        #[primary_key]
+        fn by_key(&self) -> SingletonKey {
+            SingletonKey {}
+        }
+    }
+
+    impl Default for NetworkName {
+        fn default() -> Self {
+            NetworkName {
+                name: String::from("psibase"),
+            }
+        }
+    }
+
+    #[table(record = "WebContentRow", index = 1)]
+    struct WebContentTable;
+
+    #[action]
+    #[allow(non_snake_case)]
+    fn networkNameChanged(name: String) {
+        NetworkNameTable::new()
+            .put(&NetworkName { name: name.clone() })
+            .unwrap();
+
+        Wrapper::emit().history().networkNameChanged(name);
+    }
+
+    #[event(history)]
+    pub fn networkNameChanged(name: String) {}
+
+    struct Query;
+
+    #[Object]
+    impl Query {
+        async fn network_name(&self) -> async_graphql::Result<String, async_graphql::Error> {
```

Comment:
Seems like this can just return a string, no?

Reply:
Marked as resolved but no action here either... when you mark as resolved if you're not going to do anything it would be helpful to leave a comment explaining your thinking

Reply:
same here